web/components/roomConfig/GenericWebhookConfig.tsx

```diff
@@ -83,14 +68,12 @@ const ConnectionConfiguration: FunctionComponent<{
                 }}
             />
             <p> See the <a target="_blank" rel="noopener noreferrer" href={DOCUMENTATION_LINK}>documentation</a> for help writing transformation functions </p>
-        </div>}
-
-        <div className={style.buttonSet}>
-            { canEdit && <Button onClick={onSaveClick}>{ existingConnection ? "Save" : "Add webhook"}</Button>}
-            { canEdit && existingConnection && <Button intent="remove" onClick={onRemoveClick}>Remove webhook</Button>}
-        </div>
-
-    </div>;
+        </InputField>
+        <ButtonSet>
+            { canEdit && <Button>{ existingConnection ? "Save" : "Add webhook" }</Button>}
```

Comment:
(Are you testing your changes? :eyes:)

For this to submit the form it must become a `submit` button.

```suggestion
            { canEdit && <Button type="submit">{ existingConnection ? "Save" : "Add webhook" }</Button>}
```

Reply:
Fixed, and yep now tested.